pocs/mount/bisque.py

```diff
@@ -183,6 +184,11 @@ def slew_to_target(self):
                     'dec': mount_coords[1],
                 })
                 success = response['success']
+                if success:
+                    self.status()
+                    while self.is_slewing:
```

Comment:
I'm confused by this code. self.status() returns a dictionary, which is ignored. Why bother making the status calls?

Reply:
`status` calls `_update_status`, which sets a number of attributes based on that status (`is_parked`, `is_slewing`, etc).

Which actually goes back to the question of whether or not we use `status` for any decision logic.